src/example/EASPortal.sol

```diff
@@ -84,7 +84,7 @@ contract EASPortal is AbstractPortal {
     revert NoBulkRevocation();
   }
 
-  function _getAttester() public view override returns (address) {
+  function _attester() internal view override returns (address) {
```

Comment:
@0xEillo don't we want this information to be publicly available?

Reply:
Yes it should actually